packages/chevrotain/src/parse/grammar/lookahead.ts

```diff
@@ -613,13 +635,11 @@ export function lookAheadSequenceFromAlternatives(
 export function getLookaheadPathsForOr(
   occurrence: number,
   ruleGrammar: Rule,
-  k: number,
-  orProd?: Alternation
-): LookAheadSequence[] {
+  k: number
+): LookaheadSequence[] {
   const visitor = new InsideDefinitionFinderVisitor(
     occurrence,
-    PROD_TYPE.ALTERNATION,
-    orProd
```

Comment:
again guessing this `orProd` param is some legacy that is no longer used?

Reply:
Yes, see my [answer above](https://github.com/Chevrotain/chevrotain/pull/1852#discussion_r979424128)

Reply:
This logic also seems to be related to that old fix (without a matching test sadly...)
- https://github.com/Chevrotain/chevrotain/commit/33acea3814b35bd34481b4e2bf343c071c9a15e8#diff-d274a502d8918e6523cad92df6f7b094921c3cc70c0b19f426d21cc0eab617ccR603